src/main/java/org/broadinstitute/consent/http/service/MetricsService.java

```diff
@@ -69,14 +68,12 @@ public DarMetricsSummary(DataAccessRequest dar) {
         this.projectTitle = dar.data.getProjectTitle();
         this.darCode =  dar.data.getDarCode();
         this.nonTechRus =  dar.data.getNonTechRus();
-        this.investigator = DarUtil.findPI(userDAO.findUserById(dar.userId));
```

Comment:
@greg.rushton Question: Do we want to remove the investigator attribute for dataset only or do we want to remove it altogether? I'm not too familiar with the exact rules of GDPR and this would remove the investigator detail from both the DatasetMetrics and DarMetrics. If we do want to remove it altogether, then this is the right move.

Reply:
@JVThomas  This does not affect the darMetrics endpoint. DarMetricsSummary is only used to create smaller objects representing dars for the dataset metrics endpoint